crates/sui-benchmark/src/bin/stress.rs

```diff
@@ -284,10 +290,24 @@ async fn main() -> Result<()> {
         // Make the client runtime wait until we are done creating genesis objects
         let cloned_config = configs;
         let cloned_gas = primary_gas;
-        let auth_clients = GatewayState::make_authority_clients(
-            &gateway_config,
-            NetworkAuthorityClientMetrics::new_for_tests(),
+        let auth_clients = make_authority_clients(
+            &gateway_config.validator_set,
+            gateway_config.send_timeout,
+            gateway_config.recv_timeout,
+            network_authority_client_metrics.clone(),
         );
+
+        let committee = make_committee(gateway_config.epoch, &gateway_config.validator_set)?;
+        let committee_store = Arc::new(CommitteeStore::new_for_testing(&committee));
+        let aggregator = Arc::new(AuthorityAggregator::new(
+            committee,
+            committee_store,
+            auth_clients.clone(),
+            AuthAggMetrics::new(&registry),
+            Arc::new(SafeClientMetrics::new(&registry)),
+            Arc::new(NetworkAuthorityClientMetrics::new(&registry)),
```

Comment:
The `NetworkAuthorityClientMetrics` seem to be initialised already once with the same `registry` on line `261` . I think this line here now makes the new initialisation of the `NetworkAuthorityClientMetrics` fail (a metric can be initialised only once under the same registry) and consequently make the benchmarks fail https://github.com/MystenLabs/sui/actions/runs/3232876746/jobs/5294017376

@longbowlu could you please have a look on this? I am more than happy to help resolving it